tests/services/leaderboard.test.js

```diff
@@ -0,0 +1,40 @@
+import { fetchLeaderboardToppers } from "@services/apiService";
+
+const API_URL = process.env.NEXT_PUBLIC_API_LINK;
+global.fetch = jest.fn();
+
+describe("fetchLeaderboardToppers", () => {
+  afterEach(() => {
+    fetch.mockClear();
+  });
+
+  it("fetches leaderboard toppers correctly", async () => {
+    const mockResponse = { leaderboardData: "mockedData" };
```

Comment:
i dont think this is the correct response.Can you please check with the api and add that data here?